WatchDogCore/WatchDog/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
```

Comment:
No, they are not, will add them to git ignore ([source](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore)).